src/test/java/org/symphonyoss/symphony/messageml/elements/form/RadioTest.java

```diff
@@ -0,0 +1,354 @@
+package org.symphonyoss.symphony.messageml.elements.form;
```

Comment:
We're moving the unit tests into a more "readable" approach. So the idea is that I could easily see which MessageML/PresentationML/Markdown is being created or checked. We plan on refactoring the already existing tests for other elements as well to apply this.

So, we don't really need the RadioModel class you created, nor the specific methods for assert (the method we can have, just make sure it's easy to understand which messageML I am sending, and which values I am expecting parsed from it.